vendor/arbiter/arbiter.cpp

```diff
@@ -3874,10 +3860,12 @@ Pool::Pool(
 #ifdef ARBITER_CURL
     curl_global_init(CURL_GLOBAL_ALL);
 
+    const json config(s.size() ? json::parse(s) : json::object());
```

Comment:
Can throw.  If you're not calling parse() to validate, then it seems kind of strange to convert to JSON just to go back to a string.